scripts/convert_access_to_config.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+import click
+import yaml
+
+
+@click.command(context_settings=dict(max_content_width=200))
+@click.argument("access_filename", default="access_log.yaml")
```

Comment:
maybe a little nicer, you can use `type=click.Path(exists=True))` here and get some fail fast, and help will tell people it ought to be a filename, and I guess you could use `type=click.Path(exists=False))` for config.yaml

Reply:
Done, though I left the `exists=False` off from the config filename, because I have found overwriting to be useful. Maybe I could add a flag that is needed to permit overwrite (though presumably I would then have to do the checking manually)?